### PR TITLE
fix: opentype.js

### PR DESCRIPTION
**Pull request by keroxp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)

### CODE REVIEW

File: types/opentype.js/index.d.ts [hunk 1]
```diff
@@ -353,15 +353,36 @@ export class Path {
...
-export interface PathCommand {
-    type: string;
-    x?: number;
-    y?: number;
-    x1?: number;
-    y1?: number;
-    x2?: number;
-    y2?: number;
-}
+export type PathCommand =
```
Comment (danmarshall) [inline types/opentype.js/index.d.ts:356]: Probably need to add other commands: H, V, S, T, A

**Review by danmarshall — CHANGES_REQUESTED** [review]
Can you add other path commands:  H, V, S, T, A ?
https://www.w3schools.com/graphics/svg_path.asp

**Comment by keroxp** [comment]
@danmarshall Thanks for reviewing. It seems that opentype.js uses only those 5 path commands for drawing.

https://github.com/opentypejs/opentype.js/blob/master/src/path.js#L201

**Review by danmarshall — APPROVED** [review]

**Comment by danmarshall** [comment]
Thanks for reviewing my review 🤓

**Comment by danmarshall** [comment]
Thanks for reviewing my review 🤓

**Comment by keroxp** [comment]
Ready to merge

**Pull request merged**